modules/030-cloud-provider-gcp/images/cloud-data-discoverer/main.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+
+	"github.com/alecthomas/kingpin"
+	"github.com/sirupsen/logrus"
+
+	cloud_data "github.com/deckhouse/deckhouse/go_lib/cloud-data"
+	"github.com/deckhouse/deckhouse/go_lib/cloud-data/app"
+)
+
+var (
+	credsFile = ""
+	zones     = make([]string, 0)
+	project   = ""
+)
+
+func gcpFlags(kpApp *kingpin.Application) {
+	kpApp.Flag("credentials-file", "File with gcp credentials").
+		Envar("GCP_CREDENTIALS_FILE").
+		Required().
+		StringVar(&credsFile)
+
+	kpApp.Flag("zone", "Zones for discovery").
+		Envar("GCP_ZONES").
+		Required().
+		StringsVar(&zones)
+
+	kpApp.Flag("project", "Zones for discovery").
+		Envar("GCP_ZONES").
```

Comment:
Wrong comment and env var. It should contain something like `GCP_PROJECT`

Reply:
Done